test/campaigns.js

```diff
@@ -1,43 +1,189 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
+const nockHelper = require('./helpers/nock_helper')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
+const hubspot = () =>
+  new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'fake-token',
+  })
+const appId = process.env.APP_ID || 23456
 
 describe('campaigns', function() {
   describe('get', function() {
+    beforeEach(
+      nockHelper.mockOauthEndpoint('/email/public/v1/campaigns', {
+        campaigns: [],
+      }),
+    )
+    afterEach(nockHelper.resetNock)
+
     it('Should return campaign IDs with recent activity associated with the portal', function() {
-      return hubspot.campaigns.get().then(data => {
```

Comment:
Small suggestion: For the line above, can we make this more explicit to communicate that this endpoint "returns campaigns IDs for recent email activity" or something similar? I think it's worth specifying that this endpoint returns email events. Which is the name of the API in the docs "Email Events API" bu the path `/email/public/v1/campaigns` IMHO doesn't communicate this very well.

Reply:
Done in 8099b39aeee49690d09b2ea1d461def7415fb1e9